src/components/common/ProductBox/ProductBox.js

```diff
@@ -11,9 +11,10 @@ import {
 import { faStar as farStar, faHeart } from '@fortawesome/free-regular-svg-icons';
 import Button from '../Button/Button';
 
-const ProductBox = ({ name, price, promo, stars }) => (
+const ProductBox = ({ name, price, promo, stars, image }) => (
   <div className={styles.root}>
     <div className={styles.photo}>
+      <img className={styles.image} src={image} alt='bed' />
```

Comment:
alt='bed'? a co jeśli to będzie krzesło? :D
Najlepiej zastosuj konstrukcję "Furniture-[nazwa produktu]"